lib/src/kbasesearchengine/events/handler/SourceData.java

```diff
@@ -226,13 +238,22 @@ public Builder withSourceTag(final String sourceTag) {
             sourceTags.add(sourceTag);
             return this;
         }
-        
+
+        /** Add a public or private flag of the data.
```

Comment:
should probably be `for the data`
note the value of the default

Reply:
Done